app/models/post.rb

```diff
@@ -27,7 +27,7 @@ def to_s
     title
   end
 
-  def printLimit
+  def print_limit
```

Comment:
@jforberg 
Got any good solution for a method like this?
Rules
  - `limit == 1, recLimit == 1    => 1 (x)`
  - `limit == 3, recLimit == 4    => 3-4`
  - `limit == 0, recLimit == 0    => *`
  - `limit == 0, recLimit == 10 => 10`